README.md

```diff
@@ -89,19 +89,22 @@ parser.parse(
    c:Tom a c:Cat.
    c:Jerry a c:Mouse;
            c:smarterThan c:Tom.`,
-  (error, quad, prefixes) => {
+  (error, quad) => {
     if (quad)
       console.log(quad);
     else
-      console.log("# That's all, folks!", prefixes);
+      console.log("# That's all, folks!");
   });
 ```
 The callback's first argument is an optional error value, the second is a quad.
 If there are no more quads,
-the callback is invoked one last time with `null` for `quad`
-and a hash of prefixes as third argument.
+the callback is invoked one last time with `null` for `quad`.
 <br>
-Pass a second callback to `parse` to retrieve prefixes as they are read.
+In case you would also like to process prefixes, you can instead pass an object containing multiple callbacks.
+The callback to retrieve the quads is called `onQuad`.
+The callback to also retrieve prefixes as they are read is called `onPrefix`.
+The first argument is the prefix, the second is the IRI.
+There is also a third callback called `onComment` taking only one `comment` argument.
```

Comment:
Thanks for correcting this. For completeness could you add examples as these are usually easier for developers to parse than text descriptions.

Please also note that `onComment` is available when `comment: true` is set and include this in the example.

Reply:
onComment is enabled from the moment you set the callback. No need for another constructor parameter here. This is different in the StreamParser.

I’m afraid the examples will get a bit much. The simplest example is written out in code. For more advanced features (prefixes and comments) you need to read the text?

Reply:
No need to worry, I've done some cleanup :)